setup.py

```diff
@@ -254,6 +254,11 @@ def compile_KLU():
             "pre-commit",  # For code style checking
             "black",  # For code style auto-formatting
         ],
+        "jax": [
+            "jax",
+            "jaxlib",
+        ],
```

Comment:
We rely on a particular version of these libraries -

https://github.com/pybamm-team/PyBaMM/blob/02c56a2fea4bfb094b220609d0a34042d47340a6/pybamm/util.py#L25-L26

Reply:
Specified them.